platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,81 @@
+# Platform Design Proposal Process
+
+This document describes the process for creating and reviewing platform design proposals.
+
+The status of all active design proposals can be viewed
+at [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project board.
+
+---
+
+## Creating A Proposal Draft
+
+All proposals must be presented in the form of a PR containing the proposal in a subdirectory
+of `platform-sdk/docs/proposals/`.
+
+1. Create an issue in the github repository for the creation of the proposal.
+2. Create a new branch for the proposal.
+3. Copy the template `template.md` into a subdirectory of `platform-sdk/docs/proposals/` with the name of the
+   proposal.
+4. Fill out the template with the details of the proposal, removing any irrelevant sections.
+    * Images and other files may be included in the proposal directory and linked from the proposal.
+5. Submit a draft PR with the proposal documents.
+6. Solicit feedback by stakeholders and other design collaborators.
+7. When the proposal is ready for a vote, mark the PR as ready for review.
+
+Any source code that was developed as part of the design process may be included in the subdirectory of the proposal
+but not merged into the source code of the platform.
+
+While in this draft phase the proposal may be updated and modified as needed. It is best practice to solicit review from
+stakeholders to surface any potential change requests while the proposal is in the draft phase. All grammar and spelling
+errors should be caught during this phase.
+
+---
+
+## Voting on A Proposal
+
+When a proposal is ready for a vote, the PR is marked as ready for review. The proposal must stay in the Voting state
+for 3 business days and cannot be altered during this time.
```

Comment:
Is there a definition of business days? Are holidays included? SL Company holidays? What about outside contributors who eventually start to participate, how do they know? Do we just say 3 days excluding Saturday and Sunday?

Reply:
Good question. I think we should include SL holidays. People should not feel pressured to do a rush review because one of the 3 business days is a holiday, or feel that they need to work on the holiday to get their vote in. Perhaps SL can publish a company holiday calendar somewhere when external contributors become a factor.